ironfish-rust/src/transaction/mod.rs

```diff
@@ -422,19 +424,40 @@ impl Transaction {
         // guarantee they are part of this transaction, unmodified.
         let mut binding_verification_key = ExtendedPoint::identity();
 
+        // Batch proof verifiers
+        let mut spends_verifier = Verifier::<Bls12>::new();
```

Comment:
I don't think it has to be in this PR, but I think it would be nice to replace this function with a call to batch_verify_transactions if possible, so that we only have one instance of the security-sensitive verification logic.

Reply:
Agreed. I'll have to play with this one a bit since I'll have to change some ownership stuff around that wasn't immediately obvious